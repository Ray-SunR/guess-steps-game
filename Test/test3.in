game 37 10 test3A.txt test3B.txt
quit
