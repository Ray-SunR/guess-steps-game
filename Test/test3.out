Total is 37, max deduction is 10
Player A's move
Total is 33
Player B's move
Total is 23
Player A's move
Total is 22
Player B's move
Total is 15
Player A's move
Total is 8
Player B's move
Total is 1
Player A's move
Total is -2
Player A wins
Score is
A 1
B 0
